src_assets/common/assets/web/pin.html

```diff
@@ -8,10 +8,11 @@
 <body id="app">
   <Navbar></Navbar>
   <div id="content" class="container">
-    <h1 class="my-4">PIN Pairing</h1>
+    <h1 class="my-4 text-center">PIN Pairing</h1>
     <form action="" class="form d-flex flex-column align-items-center" id="form">
       <div class="card flex-column d-flex p-4 mb-4">
-        <input type="text" pattern="\d*"  placeholder="PIN" id="pin-input" class="form-control my-4" />
+        <input type="number" pattern="\d*"  placeholder="PIN" id="pin-input" class="form-control mt-2" />
+        <input type="text" placeholder="Name (optional)" id="name-input" class="form-control my-4" />
```

Comment:
Your call. It wasn't required before, so I didn't want to enforce it. If it's going to be required, I think the field name should better indicate that they're coming up with a name for the device. Maybe "Name this device" or "Set device name"?

Reply:
I think it's important to have an identifiable name as user's aren't going to be able to differentiate based on a uuid.

Ideally we could also get the client type during the pair process, but I'm not sure that's possible at the moment.

I think "Device Name" is okay to use as the name of the field.

Reply:
It might also be nice to track/display when the client last connected and if there's an active session. But that seems like another PR for another day.

For now I'll make the field required and labeled "Device Name".

Reply:
Agreed!

I'd like to see these items tracked in the future.

- Last known IP address
- Last known network type (LAN or WAN)
- Moonlight Client (e.g. Android, QT, iOS, etc.)
- Last known moonlight version
- Device Name (provided by Moonlight, not sure if feasible... but e.g. Nvidia Shield)
- Last session start and end times
- Last session application launched

Reply:
I don't love the idea of making device name required. I'd much rather feed that info into the pairing process from the Moonlight side and allow the user to optionally adjust the name via this UI.

On the Moonlight side, it should be pretty easy to send the client's hostname across during the pairing process. We will run into some issues related to the privacy restrictions [on iOS](https://developer.apple.com/documentation/uikit/uidevice/1620015-name), but that's why we keep the optional field around as an override.

Reply:
If Moonlight can send over enough details, then the name is probably not even necessary.

This is what Plex does. None of these fields can be modified, they are all collected from the client, and everything is pretty easy to identify.

![image](https://github.com/LizardByte/Sunshine/assets/42013603/cac0acf2-0ff9-4015-a4a2-2520990b361d)

The only issue I see is some of these can change after the pairing process. i.e. the client version (and IP address if we're going to track that somehow).

Reply:
It may still be helpful to optionally set a client name during pairing. 

I suspect most users don't name their devices at the OS level, which can make the Moonlight-provided name somewhat meaningless without more info. Tracking clients this way could be especially difficult if you're using Sunshine to share your PC with someone else (remote "local" co-op, for example). Personally, I would rather set the client name myself than explain to a friend that they need to set a name in Moonlight (or worse, their device's hostname).

Also, I'd expect (well, "hope" lol) that this PR will get approved before the required additions to Moonlight are implemented. So a name field is still needed (optional or not) for now.